docs/http_api.md

```diff
@@ -0,0 +1,284 @@
+# Update Config
+
+Updates the device's configuration.
+
+## Request Destination
+
+* **Method**: POST
+* **URL**: `https://cloud4rpi.io/api/devices/{token}/config`
+
+### URL Parameters
+
+Name      | Description
+--------- | -----------------------
+token     | The [device](https://cloud4rpi.io/devices) token
+
+### Headers
+
+```
+Content-Type: application/json
+```
+
+## Request Body
+
+```json
+{
+    "ts": datetime_isoformat,
+    "payload": {
+        [
+            {"name": name, "type": type},
+            {"name": name, "type": type},
+            // ...
+            {"name": name, "type": type}
+        ]
+    }
+}
+```
+or
+```json
+[
+    {"name": name, "type": type},
+    {"name": name, "type": type},
+    // ...
+    {"name": name, "type": type}
+]
+```
+
+### Parameters
+
+Name               | Description   | Type   | Possible Values
+------------------ | ------------- | ------ | ----------------------------
+datetime_isoformat | Timestamp     | string | Time in **ISO 8601** format.
+name               | Variable name | string | Any string
+type               | Variable type | string | "bool", "numeric" or "string"
+
+
+## Example
+
+The following example creates two variables on the device:
+
+* **Temperature** of the numeric type.
+* **LEDOn** of the boolean type.
+
+
+```bash
+export DEVICE_TOKEN=your_device_token
```

Comment:
Generalize token variable names ('device_token' in all code below). Remove 'your' prefix